generate_postmortem.sh

```diff
@@ -302,6 +302,7 @@ fi
 #============================================== autodetect namespaces ============================================
 if [[ $AUTO_DETECT -eq 1 ]]; then
     NS_LISTING=`kubectl get ns 2>/dev/null | sed -e '1d' | egrep -v "kube-system|cert-manager|rook"`
+    NS_LISTING=`kubectl get ns 2>/dev/null | sed -e '1d' | egrep -v "kube-system|certmanager|rook"`
```

Comment:
The pipe (|) in the egrep regular expression means a logic OR, i.e. match kube-system OR cert-manager OR rook.
Here we just need to add one more OR certmanager to the existing line 304, not add a new line.

Reply:
Thank you for the explanation, very helpful